docs/api/clojure/symbol.md

```diff
@@ -0,0 +1,150 @@
+
+```clojure
+(def lhs (sym/variable "data1"))
+(def rhs (sym/variable "data2"))
+(def net (sym/fully-connected "fc1" {:data (sym/+ lhs rhs) :num-hidden 128}))
+(sym/list-arguments net) ;=> ["data1" "data2" "fc1_weight" "fc1_bias"]
+```
+
+## Group Multiple Symbols
+
+To construct neural networks with multiple loss layers, we can use mxnet.sym.Group to group multiple symbols together. The following example groups two outputs:
+
+```clojure
+(def net (sym/variable "data"))
+(def fc1 (sym/fully-connected {:data net :num-hidden 128}))
+(def net2 (sym/activation {:data fc1 :act-type "relu"}))
+(def out1 (sym/softmax-output {:data net2}))
+(def out2 (sym/linear-regression-output {:data net2}))
+(def group (sym/group [out1 out2]))
+(sym/list-outputs group)
+;=> ["softmaxoutput0_output" "linearregressionoutput0_output"]
+```
+
+## Serialization
+ There are two ways to save and load the symbols. You can use the mxnet.Symbol.save and mxnet.Symbol.load functions to serialize the Symbol objects. The advantage of using save and load functions is that it is language agnostic and cloud friendly. The symbol is saved in JSON format. You can also get a JSON string directly using mxnet.Symbol.toJson. Refer to API documentation for more details.
+
+ The following example shows how to save a symbol to a file, load it back, and compare two symbols using a JSON string. You can also save to S3 as well
+
+```clojure
+(def a (sym/variable "a"))
+(def b (sym/variable "b"))
+(def c (sym/+ a b))
+(sym/save c "symbol-c.json")
+(def c2 (sym/load "symbol-c.json"))
+(= (sym/to-json c) (sym/to-json c2)) ;=>true
+```
+
+
+## Executing Symbols
+
+To execute symbols, first we need to define the data that they should run on. We can do it by using the bind method, which accepts device context and a dict mapping free variable names to NDArrays as arguments and returns an executor. The executor provides forward method for evaluation and an attribute outputs to get all the results.
+
+```clojure
+(def a (sym/variable "a"))
+(def b (sym/variable "b"))
+(def c (sym/+ a b))
+
+(def ex (sym/bind c {"a" (ndarray/ones [2 2]) "b" (ndarray/ones [2 2])}))
+(-> (executor/forward ex)
+    (executor/outputs)
+    (first)
+    (ndarray/->vec));=>  [2.0 2.0 2.0 2.0]
+```
+
+We can evaluate the same symbol on GPU with different data.
+_To do this you must have the correct native library jar defined as a dependency_
+
+**Note In order to execute the following section on a cpu set gpu_device to (cpu)**
+
+
+```clojure
+(def ex (sym/bind c (context/gpu 0) {"a" (ndarray/ones [2 2]) "b" (ndarray/ones [2 2])}))
+```
+
+## Multiple Outputs
+
+;;To construct neural networks with multiple loss layers, we can use mxnet.sym.Group to group multiple symbols together. The following example groups two outputs:
```

Comment:
Why the ;;?
`mxnet.sym.Group`

Reply:
That's a code comment in clojure that I forgot to take out when I was porting the code example to this markdown format. Will fix